### PR TITLE
Add restrictions of incremental materialized views.

### PR DESCRIPTION
**Pull request by tatsuo-ishii**

### CODE REVIEW

File: doc/src/sgml/ref/create_materialized_view.sgml [hunk 1]
```diff
@@ -58,8 +58,79 @@ CREATE [ INCREMENTAL ] MATERIALIZED VIEW [ IF NOT EXISTS ] <replaceable>table_na
     <term><literal>INCREMENTAL</literal></term>
     <listitem>
      <para>
-      If specified, a materialized view enables incremental view maintenance.
-      You can replace only the contents of a materialized view, which based rows are changed.
+      If specified, some triggers and other objects are automatically created
+      so that the rows of the materialized view is immediately updated when
```
Comment (yugo-n) [inline doc/src/sgml/ref/create_materialized_view.sgml]: I am not sure what "other objects" refer. Is there other object created other than triggers?

I think "the rows of the materialized view is" would be "the rows of the materialized view are"

File: doc/src/sgml/ref/create_materialized_view.sgml [hunk 2]
```diff
@@ -58,8 +58,79 @@ CREATE [ INCREMENTAL ] MATERIALIZED VIEW [ IF NOT EXISTS ] <replaceable>table_na
...
+        </para>
+       </listitem>
+       <listitem>
+        <para>
+         Subqueries.
+        </para>
+        </listitem>
+       <listitem>
+        <para>
+         Aggregations other than count, sum or avg.
```
Comment (yugo-n) [inline doc/src/sgml/ref/create_materialized_view.sgml]: Now, we are also supporting min/max aggregate.

File: doc/src/sgml/ref/create_materialized_view.sgml (hunk 1 above)
Comment (tatsuo-ishii) [reply doc/src/sgml/ref/create_materialized_view.sgml]: I thought there might be an use of intermediate tables in future releases. If we do not have such a plan (at least by the end of this year), "other objects" should be removed.

File: doc/src/sgml/ref/create_materialized_view.sgml (hunk 2 above)
Comment (tatsuo-ishii) [reply doc/src/sgml/ref/create_materialized_view.sgml]: Oops. I will add min/max.

**Pull request merged**